deploy/helm/lunettes/templates/grafana/grafana-cm.yaml

```diff
@@ -697,7 +697,7 @@ data:
                       {
                         "targetBlank": true,
                         "title": "NodeYaml",
-                        "url": "/d/nodeyamllinks/nodeyaml?orgId=1&var-nodeName=${__data.fields[\"NodeName\"]}"
+                        "url": "/d/nodeyamllinks/nodeyaml?var-nodeName=${__data.fields[\"NodeName\"]}"
```

Comment:
Is there any difference in the usage of `__data.fields[\"NodeName\"]` and `__data.fields.PodUid`? It is recommended to unify the format.

Reply:
All have been unified into the format of 【 __data.fields.PodUid 】